.github/workflows/changelog.yml

```diff
@@ -76,13 +76,6 @@ jobs:
 
       # In order to validate any links in the yaml file, render the config to markdown
       - name: Render .chloggen changelog entries
-        run: make chlog-preview > changelog_preview.md
-      - name: Install markdown-link-check
-        run: npm install -g markdown-link-check
+        run: make chlog-preview 2> changelog_preview.md
```

Comment:
I noticed while running this `make chlog-preview > changelog_preview.md` on Ubuntu that the file created does not contain the changelog preview, but only the actual make code.

Reply:
Now it works :) 

![image](https://github.com/user-attachments/assets/0d5fb29a-d72d-4243-b088-75dd013b15a6)